stan/math/prim/arr/functor/map_rect_lpdf_mpi.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <vector>
+#include <set>
+
+#include <stan/math/prim/mat/fun/to_array_1d.hpp>
+#include <stan/math/prim/arr/functor/mpi_command.hpp>
+
+namespace stan {
+  namespace math {
+
+    template <typename F>
+    double
+    map_rect_lpdf_mpi(const std::vector<double>& eta,
```

Comment:
So this version doesn't parallelize at all on purpose?

Reply:
oh, we do. I just did not see an elegant way to reuse the mpi code in a clever way. As I only cared about getting my model faster to fit, I did not bother with this. So we either copy the rev code and simplify it or we find a clever way to generalize the rev to handle both cases in a smart way.

Reply:
Gotcha. Maybe we can figure something out... I'll think more on it.